### PR TITLE
Add Code Actions for Managed Dependency Version override

### PR DESCRIPTION
**Pull request by vrubezhny**
This PR adds Maven project validation for managed dependency version element values override/duplication
Also this adds Code Actions for the removal of the duplicating/overriding dependency version elements and according JUnit test cases

Depends on https://github.com/eclipse/lemminx-maven/pull/365

### CODE REVIEW

File: lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/participants/codeaction/MavenCodeActionParticipant.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import org.eclipse.lemminx.extensions.contentmodel.participants.XMLSyntaxErrorCode;
+import org.eclipse.lemminx.extensions.maven.MavenSyntaxErrorCode;
+import org.eclipse.lemminx.services.extensions.codeaction.ICodeActionParticipant;
+import org.eclipse.lemminx.services.extensions.codeaction.ICodeActionRequest;
+import org.eclipse.lemminx.settings.SharedSettings;
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.jsonrpc.CancelChecker;
+
+public class MavenCodeActionParticipant implements ICodeActionParticipant {
```
Comment (mickaelistria) [inline lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/participants/codeaction/MavenCodeActionParticipant.java:23]: See other PRs. We shouldn't group the participants into a single one.

Comment (vrubezhny) [reply lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/participants/codeaction/MavenCodeActionParticipant.java]: Updated

**Review by mickaelistria — APPROVED** [review]

**Comment by vrubezhny** [comment]
@mickaelistria  Updated Project Validation, Code Action and JUnit test with all 3 quickfixes

**Review by mickaelistria — APPROVED** [review]
Looks good. Please merge.

**Pull request merged**